nsqadmin/static/js/views/channel.hbs

```diff
@@ -92,6 +93,24 @@
                 <td class="bold rate" target="{{rate "topic" node topic_name ""}}"></td>
             {{/if}}
             <td>{{commafy client_count}}</td>
+            {{#if delivery_msg_count}}
+            <td>
+                <div class="popup" data-id="popup-{{node}}">
+                    <div class="progress" style="width: 120px; height: 20px; border-right: .01em solid #777; border-left: .01em solid #777; border-top: .01em solid #777; border-bottom: .01em solid #777">
+                        <div class="progress-bar progress-bar-success" style="background-color:rgb(221,255,221); width: {{floatToPercent zone_local_percentage}}%">
+                            <span class="sr-only">{{floatToPercent zone_local_percentage}}%</span>
+                        </div>
+                        <div class="progress-bar progress-bar-warning progress-bar-striped" style="background-color:rgb(254,254,194); width: {{floatToPercent region_local_percentage}}%">
+                            <span class="sr-only">{{floatToPercent region_local_percentage}}%</span>
+                        </div>
+                        <div class="progress-bar progress-bar-danger" style="background-color: white; width: {{floatToPercent global_percentage}}%">
+                            <span class="sr-only">{{floatToPercent global_percentage}}%</span>
+                        </div>
+                    </div>
+                    <span class="popuptext" id="popup-{{node}}"><span style="background-color:rgb(221,255,221); color:#4b4b4b">{{floatToDecimalPercent zone_local_percentage}}%</span> | <span style="background-color:rgb(254,254,194); color:#4b4b4b">{{floatToDecimalPercent region_local_percentage}}%</span> | <span style="color:#4b4b4b">{{floatToDecimalPercent global_percentage}}%</span></span>
```

Comment:
It's probably worth putting these in `base.css` as classes `bg-zone-local`, `bg-region-local` and `bg-global-zone` to this reads a little clearer. (can be re-used on the progress bar too)